### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by YouKnowMe2**

### CODE REVIEW

**Review by TenTraicion — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -122,6 +122,37 @@ <h2>A project for first-time contributions</h2>
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+        
+            <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (hira-saha) [inline index.html:126]: Dear committer, please Change "Template make a copy" to your user name

File: index.html [hunk 2]
```diff
@@ -122,6 +122,37 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://ray.so/" target="_blank" title="Third resource">
+                  Creating Code To Image
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (hira-saha) [inline index.html:155]: change "end template" to your user name and commit again.

**Review by hira-saha — CHANGES_REQUESTED** [review]
please make changes and commit.

**Review by hira-saha — APPROVED** [review]

**Comment by hira-saha** [comment]
@YouKnowMe2 thanks for your PR. Your card will be live within 2-3 minutes.

**Pull request merged**